projects/shikshalokam/src/lib/core-module/components/side-nav/side-nav.component.html

```diff
@@ -0,0 +1,28 @@
+<mat-sidenav-container fullscreen>
```

Comment:
Why is this container here???

Reply:
if i place inside the <mat-sidenav-container> in portal and <mat-sidenav> in library it is not working. that is why i kept in library only